python_scripts/shellies_discovery.py

```diff
@@ -3230,6 +3268,65 @@ def mqtt_publish(topic, payload, retain, json=False):
 
     mqtt_publish(config_topic, payload, retain, json=True)
 
+# color lights
+for light_id, light_options in color_lights.items():
+    if device_config.get(f"light-{light_id}-name"):
+        light_name = device_config[f"light-{light_id}-name"]
+    else:
+        light_name = f"{device_name} Light {light_id}"
+
+    unique_id = f"{dev_id}-light-{light_id}".lower()
+    config_topic = f"{disc_prefix}/light/{dev_id}-{light_id}/config".encode(
+        "ascii", "ignore"
+    ).decode("utf-8")
+
+    payload = {
+        KEY_SCHEMA: VALUE_TEMPLATE,
+        KEY_NAME: light_name,
+        KEY_COMMAND_TOPIC: light_options[KEY_COMMAND_TOPIC].format(light_id=light_id),
+        KEY_STATE_TOPIC: light_options[KEY_STATE_TOPIC].format(light_id=light_id),
+        KEY_AVAILABILITY: availability,
+        KEY_COMMAND_ON_TEMPLATE: (
+            "{^turn^:^on^"
+            "{%if red is defined and green is defined and blue is defined%}"
+            ",^mode^:^color^,^red^:{{red}},^green^:{{green}},^blue^:{{blue}}"
+            "{%endif%}"
+            "{%if brightness is defined%}"
+            ",^gain^:{{brightness|float|multiply(0.3922)|round}},"
+            "^brightness^:{{brightness|float|multiply(0.3922)|round}}"
+            "{%endif%}"
+            "{%if effect is defined%}"
+            "{%if effect==^Meteor Shower^%},^effect^:1"
+            "{%elif effect==^Gradual Change^%},^effect^:2"
+            "{%elif effect==^Flash^%},^effect^:3"
+            "{%else%},^effect^:0"
+            "{%endif%}"
+            "{%else%},^effect^:0"
+            "{%endif%}"
+            "{%if color_temp is defined%},^mode^:^white^,"
+            "^temp^:{{(1/(color_temp|float))|multiply(1000000)|round}}"
+            "{%endif%}}"
+        ),
+        KEY_COMMAND_OFF_TEMPLATE: "{^effect^:0,^turn^:^off^}",
+        KEY_STATE_TEMPLATE: "{%if value_json.ison%}on{%else%}off{%endif%}",
+        KEY_BRIGHTNESS_TEMPLATE: "{%if value_json.ison and value_json.mode==^white^%}{{value_json.brightness|float|multiply(2.55)|round(0)}}{%else%}{{value_json.gain|float|multiply(2.55)|round(0)}}{%endif%}",
+        KEY_RED_TEMPLATE: "{%if value_json.ison and value_json.mode==^color^%}{{value_json.red}}{%else%}0{%endif%}",
+        KEY_GREEN_TEMPLATE: "{%if value_json.ison and value_json.mode==^color^%}{{value_json.green}}{%else%}0{%endif%}",
+        KEY_BLUE_TEMPLATE: "{%if value_json.ison and value_json.mode==^color^%}{{value_json.blue}}{%else%}0{%endif%}",
+        KEY_EFFECT_LIST: ["Aus", "Meteor Shower", "Gradual Change", "Flash"],
+        KEY_EFFECT_STATE_TOPIC: light_options[KEY_STATE_TOPIC].format(light_id=light_id),
+        KEY_EFFECT_VALUE_TEMPLATE: "{%if value_json.effect==1%}Meteor Shower{%elif value_json.effect==2%}Gradual Change{%elif value_json.effect==3%}Flash{%else%}Aus{%endif%}",
```

Comment:
`effect_value_template` is not a part of the template schema https://www.home-assistant.io/integrations/light.mqtt/#template-schema

Reply:
Where did you get the error message from?
You can find `effect_value_template` in the schema here:

https://www.home-assistant.io/integrations/light.mqtt/#effect_value_template